src/components/app/app.js

```diff
@@ -2,32 +2,23 @@ import template from "./app.html";
 
 import { DrawerComponent } from "../drawer/drawer";
 import { ToolbarComponent } from "../toolbar/toolbar";
-import { AddTransactionComponent } from "../add-transaction-dialog/add-transaction-dialog";
+import { ContentComponent } from "../content/content";
```

Comment:
maybe it is better to call it not a `ContentComponent` but something more descriptive ?